lib/validators/specResolver.ts

```diff
@@ -453,10 +453,12 @@ export class SpecResolver {
         this.reportError,
         this.docsCache
       )
+      const result2: any = result
```

Comment:
Please, don't use `any`. Use `unknown` if possible.

Could we have a better name instead of `result2` ?